### DISCUSSION TITLE
XCode error when running Uno.Check

### DISCUSSION THREAD

**Discussion by Juriyx** [unoplatform/uno#17859] [Q&A] [answered]
Hello. I get this error when I run Uno.Check on macOS:
`xcodebuild: error: The directory /Users/runner/work/1/s does not contain an Xcode project.`

Why does it check the sources directory and what can be the reason?

**Comment by Juriyx** [comment]
@agneszitte Maybe you can help?

**Reply by spouliot** [reply]
This is not a `uno-check` error and is _normal_ (expected). 

`xcodebuild` has to be called to verify that it _works_, specifically that it does not fail because the EULA has not been accepted. That would lead to hard to diagnose errors when building apps for iOS or macOS.

IOW the fact that it fails to find an Xcode project means the EULA has been accepted.

Ideally that output would be hidden but IIRC that caused a different issue...

**Reply by spouliot** [reply, marked as answer]
https://github.com/unoplatform/uno.check/issues/279
